README.md

```diff
@@ -1,9 +1,52 @@
-# Northcoders News API
+# 📰 Northcoders News API
 
-We'll have two databases in this project: one for real-looking dev data, and another for simpler test data.
+Hosted: https://nc-news-im.onrender.com/api
+
+# 🤖 About
+
+The nc-news API is a PostgreSQL database designed to access application data programmatically that mimics a real world backend service. It will provide the information to the front-end architecture. The server is able to:
+
+- Respond with a list of available endpoints
+- Respond with a list of topics, articles and users
+- Respond with a single article or article comments based on id
+- Allows articles to be filtered and sorted
+- Add and delete comments
+- Update an article by id
+
+The server has been built using Test-Driven-Development to ensure software stability.
+
+# 📝 How to Build
```

Comment:
How to **run** locally, build implies something a bit different. Build refers to the process of packaging the API so that it can be deployed to a production environment, which isn't what these instructions are for.